### PR TITLE
a few documentation cleanups

### PR DESCRIPTION
**Pull request by tim-fiola**
Added a line to do a nautobot server migration after editing the nautobot_config.py file as I've had to do that each time in order to get it to work

### CODE REVIEW

**Comment by smk4664** [comment]
Editing the Nautobot_config.py should not require running the migrate command.  I am interested if something else is happening when you run the migrate commad?  It should just take a restart of the Worker and the Server.

File: docs/chat_setup/chat_setup.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ PLUGINS_CONFIG = {
     }
 }
 ```
+After editing the `nautobot_config.py` file, as the `nautobot` user, run the `nautobot-server migrate` command.
 
 As a sudo-enabled user, restart the `nautobot` and `nautobot-worker` process after updating `nautobot_config.py`.
```
Comment (jvanderaa) [inline docs/chat_setup/chat_setup.md:93]: Combine into one if we believe the update is needed. I see this as more explicit than previously which is good, but we ask for the restart process twice.

**Review by jvanderaa — CHANGES_REQUESTED** [review]
Thank you for the update!

**Comment by tim-fiola** [comment]
Sorry folks. I should have opened this as a bug instead of a doc change.  Let me close this out and document what I see when I don't do the migration.

**Pull request closed without merging**